pkg/sideinputs/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+func CheckFileExists(fileName string) bool {
+	_, err := os.Stat(fileName)
+	// check if err is "file not exists"
+	if os.IsNotExist(err) {
+		return false
+	} else {
+		return true
+	}
+}
+
+func UpdateSideInputStore(filePath string, value []byte) error {
+	if !CheckFileExists(filePath) {
+		f, err := os.Create(filePath)
+		if err != nil {
+			return fmt.Errorf("failed to create side-input-%s : %w", filePath, err)
+		}
+		err = f.Close()
+		if err != nil {
+			return err
+		}
+	}
+	f, err := os.Create(filePath)
+	if err != nil {
+		return fmt.Errorf("failed to create side-input file: %w", err)
+	}
+	defer f.Close()
+	_, err = f.Write(value)
+	if err != nil {
+		return fmt.Errorf("failed to write side-input-%s : %w", filePath, err)
+	}
+	return nil
+}
+
+func FetchSideInputStore(filePath string) ([]byte, error) {
```

Comment:
add go doc comment
also if it's only used in the test... add a comment too..

Reply:
Done